src/js/components/channels/twilio-channel-content.jsx

```diff
@@ -1,25 +1,34 @@
-import React, { Component, PropTypes } from 'react';
-import isMobile from 'ismobilejs';
+import React, { Component } from 'react';
+
+import { ReactTelephoneInput } from '../../lib/react-telephone-input';
 
 export class TwilioChannelContent extends Component {
-    static contextTypes = {
-        settings: PropTypes.object
+    state = {
+        numberValid: false
     }
 
-    render() {
-        const {settings} = this.context;
-        const {phoneNumber} = this.props;
-        const styleOverride = settings.linkColor ? {
-            color: `#${settings.linkColor}`
-        } : null;
 
-        if (isMobile.any) {
-            return <a href={ `sms:${phoneNumber}` }
-                      style={ styleOverride }>
-                       { phoneNumber }
-                   </a>;
+    onNumberValid = () => {
+        this.setState({
+            numberValid: true
+        });
+    }
+
+    onNumberInvalid = () => {
+        this.setState({
+            numberValid: false
+        });
+    }
 
-        }
-        return <span className='channel-content-value'>{ phoneNumber }</span>;
+    render() {
+        return <div>
+                   <ReactTelephoneInput ref='telInput'
+                                        defaultCountry='ca'
```

Comment:
we talked about having it configurable in some previous PRs, don't know what was the outcome of that discussion